OHM-Trade-Agent-v1/deploy/analytics/run-gated-stage.sh

```
@@ -0,0 +1,102 @@
+}
+trap cleanup EXIT
+
+case "$STAGE" in
+  prepare)
+    for unit in \
+      opip-learning-sync.timer \
+      opip-learning-capture.timer \
+      opip-learning-outcomes.timer; do
+      if systemctl cat "$unit" >/dev/null 2>&1; then
+        systemctl disable --now "$unit"
+        ! systemctl is-active --quiet "$unit"
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)